packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/routes.jsx.hbs

```diff
@@ -18,20 +24,62 @@ const fallback = <Skeleton height="75vh" width="100%" />
 
 // Create your pages here and add them to the routes array
 // Use loadable to split code into smaller js chunks
+{{#if answers.project.hybrid}}
+const Home = loadable(() => import('@salesforce/retail-react-app/app/pages/home'), { fallback })
+{{else}}
 const Home = loadable(() => import('./pages/home'), {fallback})
+{{/if}}
 const MyNewRoute = loadable(() => import('./pages/my-new-route'))
+{{#if answers.project.hybrid}}
+const PageNotFound = loadable(() => import('@salesforce/retail-react-app/app/pages/page-not-found'))
+{{/if}}
 
 const routes = [
     {
         path: '/',
         component: Home,
         exact: true
     },
+    {{#if answers.project.hybrid}}
+    {
+        path: '/home',
+        component: Home,
+        exact: true
+    },
+    {{/if}}
     {
         path: '/my-new-route',
         component: MyNewRoute
     },
+    {{#if answers.project.hybrid}}
+    // remove routes from the base template that we want to go to SFRA
+    ..._routes.filter((route) => {
+        return !(route.path === '/cart' || route.path === '/checkout' || route.path === '*')
+    }),
+    {
+        path: '*',
+        component: withRouter((props) => {
+            const { location } = props
+            const urlParams = new URLSearchParams(location.search)
+
+            useEffect(() => {
+                if (!urlParams.has('redirected')) {
+                    urlParams.append('redirected', '1')
+                    history.push({
+                        pathname: location.pathname,
+                        search: urlParams.toString()
+                    })
+                }
+            }, [location.pathname])
```

Comment:
This useEffect is different than what we have in our docs. I find that using `window` as the docs suggest leads to errors where `window` is undefined.

Once we verify this updated code works on a hybrid site we probably will want to update the doc with this code.

Reply:
Updated this. Using window.location.href is fine inside the useEffect as that runs client side. We want the useEffect here as that guarantees a hard refresh (which history.push does not do)

Continuing to use [location.pathname] as the useEffect's dependency works to avoid server side shenanigans where window is not defined while also preventing the useEffect from triggering multiple times on redirect (we do not want this as this leads to duplicate calls in plugin_slas)

Reply:
This has been tested to be working on https://test.phased-launch-testing.com/ (navigate to the cart page, verify there is only 1 redirect and that you are sent to SFRA)